src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -99,7 +105,8 @@ const ChatItemMyTextV2 = ({
             previewBackgroundColor="transparent"
             style={{flex: 1}}
             actions={contextMenuActions}
-            onPress={(e) => onContextMenuPressed(e, data, chatType)}>
+            onPress={(e) => onContextMenuPressed(e, data, chatType)}
+            disabled={attachments.length > 1}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `disabled` prop is added to the `ContextMenu` component to disable the context menu when there are more than one attachments. This might be a design decision to avoid complexity when multiple attachments are present.
```diff
+             disabled={attachments.length > 1}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->